server/entity/Groceries.ts

```diff
@@ -0,0 +1,35 @@
+import {Entity, PrimaryGeneratedColumn, Column, BaseEntity} from "typeorm";
+import { ObjectType, Field, ID } from "type-graphql";
+
+@Entity()
+@ObjectType()
+export class Groceries extends BaseEntity {
+
+    @Field(() => ID)
+    @PrimaryGeneratedColumn()
+    id: number;
+
+    @Field(() => String)
+    @Column({
+        length: 200
+    })
+    item: string;
+
+    @Field(() => String)
+    @Column()
+    creatorid: string;
```

Comment:
Took a quick look through the code and took a quick look at the picture you provided of the UI.
Could you explain how the creatorid will be used in your shopping list plans?

Reply:
I will make use of the avatars in the 1.0.0 PR, and I thought it might also be a good idea to have the name of the creator in there. I can go over that choice in Slack as well. Better to be safe than sorry.

Reply:
Gotcha. Thanks for the clarification